src/compiler/crystal/command/format.cr

```diff
@@ -68,7 +68,7 @@ class Crystal::Command
           when .invalid_byte_sequence?
             error "'#{result.filename}' is not a valid Crystal source file", exit_code: nil
           when .bug?
-            error "there's a bug formatting '#{result.filename}', please report it including the contents of the file: https://github.com/crystal-lang/crystal/issues", exit_code: nil
+            error "there's a bug formatting '#{result.filename}', to show more information, please run:\n\n  $ crystal tool format '#{result.filename}'", exit_code: nil
```

Comment:
Why not show the backtrace for the file here? Why do you tell the user to run format on the individual file?

Reply:
Why not just print all the backtraces then? Justg a simple
```
Error: couldn't format foo.cr, please report a bug including the contents of it
<inspect_with_backtrace>

Error: couldn't format bar.cr, please report a bug including the contents of it
<inspect_with_backtrace>
```

Reply:
I explained this already: "It is hard to read multiple backtrace", and I worry to open a new issue with unrelated multiple backtraces.

Reply:
And how does making them run the formatter for each file which is broken help that?

Reply:
They will see new message "please report a bug including the contents of it" by running formatter for each file, it helps them, and I think my solution is kludge enough.

Reply:
I think we need another opinion on this.